utils/markdown-link-attributer.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-param-reassign */
+
+// Ref. https://github.com/crookedneighbor/markdown-it-link-attributes/blob/7e83cca/index.js
```

Comment:
I assume this code is not the same with `npm install markdown-it-link-attributes`, so you copy here.

If it is copies, I would just put our options here. The `markdown-loader.js` is getting complex.